scripts/app.js

```diff
@@ -281,10 +395,18 @@ function saveContentToCache(){
 }
 
 function putCacheContentToInput(){
-	var codeDA = localStorage['codeDA'] || '';
-	document.getElementById("input").value = codeDA;
+    input.value = localStorage['codeDA'] || '';
 }
 
 function isSaveKeyboardShortcut(event) {
```

Comment:
Here is an idea of implementation to prevent redondant `if` statements:

```
const userAgent = window.navigator.userAgent;

const osMapping = {
    "Linux": "Linux",
    "Mac": "Mac",
    "Win": "Windows",
    "X11": "Android"
};

const os = Object.keys(osMapping).find(key => userAgent.includes(key)) || "unknown";

return (os === "Mac" ? event.metaKey : event.ctrlKey) && event.keyCode === 83;
```